build/validate.sh

```diff
@@ -93,39 +94,24 @@ function checkDependencyViolationFinder {
 function checkWithLatestShop {
     echo "Checking module with latest $PRODUCT_NAME..."
 
-    composer config repositories.ecomodule path "$TRAVIS_BUILD_DIR/$MODULE_DIR"
-    composer update --with-all-dependencies
-    composer require "spryker-eco/$MODULE_NAME @dev" --prefer-source
-    result=$?
-
-    if [[ "$result" = 0 ]]; then
-        buildMessage="${buildMessage}\n${GREEN}$MODULE_NAME is compatible with the modules used in $PRODUCT_NAME"
-        if runTests; then
-            buildResult=0
-            checkLatestVersionOfModuleWithShop
-        fi
-    else
-        buildMessage="${buildMessage}\n${RED}$MODULE_NAME is not compatible with the modules used in $PRODUCT_NAME"
-        checkLatestVersionOfModuleWithShop
-    fi
-}
-
-function checkLatestVersionOfModuleWithShop {
     echo "Merging composer.json dependencies..."
     updates=`php "$TRAVIS_BUILD_DIR/ecoci/build/merge-composer.php" "$TRAVIS_BUILD_DIR/$MODULE_DIR/composer.json" composer.json "$TRAVIS_BUILD_DIR/$MODULE_DIR/composer.json"`
     if [[ "$updates" = "" ]]; then
         buildMessage="${buildMessage}\n${GREEN}$MODULE_NAME is compatible with the latest version of modules used in $PRODUCT_NAME"
-        return
     fi
 
-    buildMessage="${buildMessage}\nUpdated dependencies in module to match $PRODUCT_NAME\n$updates"
     echo "Installing module with updated dependencies..."
+    composer config repositories.ecomodule path "$TRAVIS_BUILD_DIR/$MODULE_DIR"
+    composer update --with-all-dependencies
```

Comment:
I'm okey to get any updates, but are we sure that core modules were tested with all of those?
Maybe it's better to include `spryker*/*` only?